### DISCUSSION TITLE
Example app broken when reverse-proxied behind HTTPS nginx

### DISCUSSION THREAD

**Discussion by brianwhu** [MasterKale/SimpleWebAuthn#498] [Troubleshooting]
Nginx proxy setting:

```
        location /fido/ {
            proxy_set_header   X-Forwarded-For $remote_addr;
            proxy_set_header   Host $http_host;
            proxy_pass         "http://127.0.0.1:8000/";
        }

```

Page loads fine. But clicking on Register gives

```
SyntaxError: Unexpected token '<', "<html>
<h"... is not valid JSON
```

**Comment by brianwhu** [comment]
Resolved by prepending "/fido" to all URL in fetch calls in public/index.html

**Reply by MasterKale** [reply]
Nginx' `proxy_pass` supports full URLs so you could probably make this work just by changing the NGINX config to this instead:

```
location /fido/ {
    proxy_set_header   X-Forwarded-For $remote_addr;
    proxy_set_header   Host $http_host;
    proxy_pass         "http://127.0.0.1:8000/fido/";
}
```

This way you wouldn't need to change anything in **public/index.html**.

**Reply by MasterKale** [reply]
https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_pass
